src/routes/index.ts

```diff
@@ -17,6 +17,7 @@ import {
   landing,
   updateLanding,
   overseasEntityQuery,
+  updateConfirmation,
```

Comment:
Can perhaps be moved to below current line 44

Reply:
Fixed